demo/lightning/image-segmentation/README.md

```diff
@@ -0,0 +1,167 @@
+# Image Segmentation Demo Code
+The code examples in this directory demonstrate how Dataflux can be used for image segmentation training alongside PyTorch Lightning. The image segmentation workload implemented here works on the [KiTS19](https://github.com/neheller/kits19) dataset which contains `210` images and their corresponding labels. The images and their labels are stored in separate directories.
```

Comment:
Do we still want the "Dataflux" naming?

Reply:
Should I change it to "...demonstrate how GCS Connector for Pytorch (previously known as Dataflux)..." or just "...GCS Connector for Pytorch" ?

Reply:
The latter sounds good to me

Reply:
Done